packages/flutter_tools/test/src/context.dart

```diff
@@ -222,15 +222,42 @@ class MockSimControl extends Mock implements SimControl {
   }
 }
 
-class MockOperatingSystemUtils extends Mock implements OperatingSystemUtils {
+class MockOperatingSystemUtils implements OperatingSystemUtils {
```

Comment:
Rename `FakeOperatingSystemUtils`.  Otherwise, it looks like a Mockito mock and makes the reader wonder how the test is getting by without the mock having been set up (`when(...).thenReturn(...)`)

/me deletes his earlier comment asking how the test was getting by without the mock having been set up.

Reply:
We have lots of MockFoos in here that aren't mockito mocks.

Reply:
(in fact they're in the majority compared to MockFoos that are mockito-related)

Reply:
heh, looks like you ended up doing this anyway...